src/metabase/driver/sql_jdbc/execute.clj

```diff
@@ -660,13 +660,26 @@
   (let [row-thunk (row-thunk driver rs rsmeta)]
     (qp.reducible/reducible-rows row-thunk canceled-chan)))
 
+(defmulti inject-remark
```

Comment:
New driver methods should be documented in `docs/developers-guide/driver-changelog.md`.

Reply:
Done, thanks!